teams/vso/teams/representatives/engineering/ADRs/using-sign-in-service.md

```diff
@@ -0,0 +1,60 @@
+** Benefits Representative Facing Tools ADR for using Sign-in Service
+
+*** Status: accepted
+
+*** Context
+
+One of the main user stories for our MVP is
+
+> As a VSOfficer: I require a secure login system to verify my accreditation and identity in order to protect Veterans’ Personal Identifiable Information (PII) and Protected Health Information (PHI).
+
+There are two different main options for login systems we can use at the time of writing this, SAML authentication and Sign-in Service. The SAML authentication is
+what is currently used on VA.gov and has a lot of existing infrastructure. Sign-in service is newer and at the time of writing this is only used for the VA mobile app
+and also has some existing infrastructure. Since we are building an MVP at this time we need to choose an authentication system that mostly works off-the-shelf as well
+as beign secure.
+
+It was discussed that one of the main UX and technical features we need for our app is for it to appear completely distinct from VA.gov and that when a VSO logs into
+our app we do NOT want them to be automatically logged into VA.gov. SAML authentication may be able to do this however at the time of writing this it is unclear if
+it CAN actually accomplish this in its current state. Sign-in service also allows us to future-proof our app since it is the reccomendend authentication system
+by the Identity team moving forward.
+
+On the topic of UX the SAML authentication and the sign-in service operate very differently. The SAML Authentication uses a modal that pops up and has various log-in
+methods such as Login.gov and ID.me. The Sign-in service makes use of a Unified Sign In Page that is currently hosted on VA.gov and requres a link to this page, then once
+the user logs in they are redirected to a URL set in the coinfiguration.
+
+*** Decision
+
+We have descided to move forward with the sign-in service for a few main reasons - 
+
+**** Either authentication option requres about the same amount of configuration and work on our end
+
+Since we will have our app on a subdomain and will need to appear visually distict from VA.gov we would need to configure the SAML authentication to work on our
+subdomain if we were to use it. The sign-in service will work in conjunction with our subdomain with a similar amount of configuration however the configuration needed
+can be performed by our team instead of an external team.
+
+**** Sign-in service is more future-proof
+
+Sign-in service is the system that the Identity team is reccomending all team adopt going forward and allows us greater flexibility going forward as more platforms
+adopt it we can possibly enable Single Sign On with some of those apps.
```

Comment:
We may want to confirm that the apps that ARF will rely upon have plans to adopt this recommendation (and when).